jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -12,6 +12,9 @@ def run(params) {
 
         env.common_params = "--outputdir ${resultdir} --tf ${params.tf_file} --gitfolder ${resultdir}/sumaform"
 
+        // Monitoring doesn't have MU channel and is reusing sle minion channel. Specify which one.
+        env.monitoring_sle_version = 'sle15sp3'
```

Comment:
is there no variable for which version of the sles minion we're using? I'd feel better if this wasn't hardcoded, especially as this is for the general pipeline and not a specific version